discovery-provider/src/api_helpers.py

```diff
@@ -33,15 +33,15 @@ def error_response(error, error_code=500):
 def success_response_backwards_compat(response_entity=None, status=200):
     response_dictionary = response_dict_with_metadata(response_entity)
     response_dictionary = {**response_dictionary, **response_entity}
-    signature, timestamp = generate_signature_and_timestamp({'data': response_dictionary['data']})
+    signature, timestamp = generate_signature_and_timestamp(response_dictionary)
```

Comment:
just to confirm, this doesn't change the response format? the 'data' key is still there right?

Reply:
yeah! in `response_dict_with_metadata()`, it creates and returns a dictionary with the metadata and data property. then, we use that response_dictionary to generate the signature. the response is still the same